corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -343,7 +343,17 @@ def _process_forms_subset(self, forms):
         if forms == "skipped":
             self._process_skipped_forms()
             return
-        for form_id in forms.split():
+        if forms.startswith(("./", "/")):
```

Comment:
`forms` could be a constant, or a filename, or a space-separated list of form IDs. I think it's OK here, but I'd tend to use a different variable for a filename.

Reply:
It requires a fair amount of boilerplate to add another parameter to this thing, so I went with overloading (and kinda strict path requirements). I know it's not pretty...

Reply:
Seems totally reasonable.